site2/docs/cookbooks-message-throttling.md

```diff
@@ -0,0 +1,86 @@
+---
+id: message-throttling
+title: Message throttling
```

Comment:
I think we only explain `Message Dispatch throttling` here. (does not contain producer throttling info).

Reply:
> I think we only explain Message Dispatch throttling here. (does not contain producer throttling info).

already changed the title to "Message dispatch throttling"